src/components/card/card.module.css

```diff
@@ -1,70 +1,72 @@
 .cardBody {
     position: relative;
     display: flex;
-    justify-content: space-evenly;
+    justify-content: space-between;
     flex-direction: column;
     align-items:center;
     width: 180px;
     height: 290px;
+    padding: 5% 5% 5% 5%;
     background-color: rgba(255, 255, 255, 0.5);
     border: 1px solid white;
     border-radius: 2px;
     text-align: center;
     top: 0;
     left: 0;
     transition: transform 1s;
+    overflow: hidden;
```

Comment:
In case the card size ends up being dynamic, we may never want the have the interior of the card to overflow because that wouldn't be physically possible. We may want to allow for an overflow given the theme is scifi which could allow for some reality bending.